doc/files/opensc.conf.5.xml.in

```diff
@@ -1902,6 +1925,14 @@ app <replaceable>application</replaceable> {
 						Extended example configuration file
 				</para></listitem>
 			</varlistentry>
+			<varlistentry>
+				<term>
+					<filename>@sysconfdir@/opensc/openssl.cnf</filename>
```

Comment:
With an empty value of `openssl_config` by default, I think this reference can be removed now.

If you want to add some *example* configuration file, then you need to add it to this PR. Preferably this should go to `docdir` then.